testcases/PowerNVDump.py

```diff
@@ -1323,6 +1325,32 @@ def runTest(self):
             if not obj.update_kernel_cmdline(self.distro, remove_args="fadump=nocma", reboot=True, reboot_cmd=True):
                 self.fail("KernelArgTest failed to update kernel args")
 
+class KernelCrash_KdumpSoftlockup(PowerNVDump):
+
+    '''
+    This test verifies kdump/fadump after inserting softlockup kernel module.
+    '''
+
+    def runTest(self):
+        self.setup_test()
+        directory_path = "/tmp/kernel-module-bug-196448"
+        # Make sure /tmp/skiboot directory does not exist
+        self.c.run_command("cd /tmp; ls -1; rm -rf %s" % directory_path)
+       
+        #clone this repo
+        r_cmd = "git clone git@github.ibm.com:sjain014/kernel-module-bug-196448.git /tmp/kernel-module-bug-196448"
+        self.c.run_command(r_cmd)
+        time.sleep(20)
+        
+        #compile kernel modeules
+        self.c.run_command("cd /tmp/kernel-module-bug-196448; make")
+        time.sleep(30)
+        
+        #Enable softlockup
+        self.c.run_command("sysctl -w kernel.softlockup_panic=1")
+
+        boot_type = self.kernel_crash(crash_type="lock")
```

Comment:
soft lock to differentiate with hard loc

Reply:
Done

Reply:
Done